### PR TITLE
post: cdktf provider docs

### PR DESCRIPTION
**Pull request by janaka**
New blog post

### CODE REVIEW

File: content/blog/Terraform-provider-docs-converted-to-CDKTF-TypeScript/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Terraform Provider Docs for CDKTF"
+date: "2023-03-25T10:14:03.000Z"
+author: "Janaka Abeywardhana"
+description: "Announcing Terraform provider docs with code blocks conversed to CDKTF"
```
Comment (cwang) [inline content/blog/Terraform-provider-docs-converted-to-CDKTF-TypeScript/index.md]:
```suggestion
description: "Announcing (unofficial) Terraform provider docs with HCL code blocks converted to CDKTF"
```

**Review by cwang — COMMENTED** [review]
Cool. On a side note, when I look through the docs generated, I wonder what `D` and `R` mean over there? Could we make the names (links are fine) better, and those pages underneath them too?
Screenshot 2023-03-27 at 09 02 10

**Comment by janaka** [comment]
Yes. Need to update the converter to create those as `data` and `resrouce`.

The slashes are technically an issue with mkdocs, not handling escape char whem genning menu names. Or the MD serialiser. Haven't found a setting for either yet. Trying to avoid a manual patch.

**Comment by janaka** [comment]
@cwang I've fixed both the issues and a few others. Refreshed docs have been published. Good to go.

https://openfabr.github.io/terraform-provider-cdk-docs/

**Comment by janaka** [comment]
There's another PR I need to push before this, so pls don't merge yet

**Comment by janaka** [comment]
@cwang ready for merge. any more comments?

**Review by cwang — APPROVED** [review]

**Pull request merged**